bug-triage/triage-workflow.md

```diff
@@ -1,6 +1,14 @@
 # Bug Triage Overview
 
-## Triage Workflow
+## Triage Outline
+
+1. Confirm that the issue is valid and filled in properly
+2. Do an initial assessment of the issue
+3. Test the issue, or ask someone else to test it
+4. Complete the assessment of the issue based on testing
+5. Hand the issue over to the relevant team(s)
```

Comment:
Some of these are potentially a bit confusing since they look quite similar, I would potentially suggest along the lines of:

1. Confirm the issue is valid and filled in properly
2. Add relevant tags
3. Reproduce the issue, or ask someone else to reproduce it
4. Assign issue to project trackers
5. Either:
   1. Close the issue
   2. Hand the issue over to the relevant team(s)

The changes to 2-4 seem to keep it brief still but be a bit clearer about the action. I added the split in 5 since there is also now a closing issue section at this point, but I'm not sure if this is the right location for the branch or if a different structure makes more sense

Reply:
This outline section now seems to be superseded by the checklist below, so it should probably be removed